lambda/__tests__/jest.setup.js

```diff
@@ -8,6 +8,7 @@ process.env.NODE_ENV = 'development';
 process.env.INCLUDE_DIGITAL_CREDENTIAL = true;
 process.env.INCLUDE_BC_SERVICES_CARD = true;
 process.env.ALLOW_BC_SERVICES_CARD_PROD = true;
+process.env.LOCAL_DEV = true;
```

Comment:
Should this be true always or only on your local?

Reply:
This is setting it just on the jest tests, which is intentional since they run in the same setup as local. It is only used to set CORS rules and this pr just added a function to handle the different file structure between local and lambda